src/login/InitResetForm/InitResetForm_actions.js

```diff
@@ -0,0 +1,23 @@
+export const DEAL_WITH_EMAIL = "DEAL_WITH_EMAIL";
```

Comment:
Is there a specific reason for this name? Could it be more descriptive and maybe related to resetting password (if needed)? Could it be the same action as the Email component?

Reply:
No hard reason, since it does not come from the backend.  The only reason is that it seems to me informative and concise: the name is DEAL_WITH _EMAIL, and it is in an InitResetForm component - that gives me enough of a clue.

The action in the Email component you refer to is ADD_EMAIL?